gallery_dl/extractor/deviantart.py

```diff
@@ -802,7 +809,7 @@ def deviations(self):
             yield from self.status(status)
 
     def status(self, status):
-        for item in status.get("items", ()):  # do not trust is_share
+        for item in status.get("items") or ():  # do not trust is_share
```

Comment:
What's the point of this ?

Reply:
See https://github.com/mikf/gallery-dl/pull/3541#discussion_r1082885666

This crashes (`TypeError: 'NoneType' object is not iterable`)
``` python
status = {"items": None}
for item in status.get("items", ()):
    pass
```

This does not
``` python
status = {"items": None}
for item in status.get("items") or ():
    pass
```

Reply:
Do I even wanna know why the Py core devs thought this was a good idea ? Surely they coulda included the `or ...` functionality in the `.get`.